src/lib/Scenes/Artwork/Components/CommercialPartnerInformation.tsx

```diff
@@ -34,6 +34,11 @@ export class CommercialPartnerInformation extends React.Component<Props> {
                 {artwork.shippingInfo}
               </Sans>
             )}
+            {artwork.priceIncludesTax && (
```

Comment:
@lilyfromseattle lets circle back and add tests here once you merge your current PR adding the test file for this component.